packages/commonwealth/server/util/entitiesProxy.ts

```diff
@@ -21,8 +33,8 @@ async function ceProxy(ceEndpoint: 'entities' | 'events', req, res) {
 
 function setupCEProxy(app: Express) {
   // using bodyParser here because cosmjs generates text/plain type headers
-  app.get('/api/ce/entities', ceProxy.bind(null, 'entities'));
-  app.get('/api/ce/events', ceProxy.bind(null, 'events'));
+  app.get('/api/ce/entities', calcCeProxyCacheKeyDuration.bind(null, 'entities'), cacheDecorator.cacheMiddleware(60*5, lookupKeyDurationInReq), ceProxy.bind(null, 'entities'));
```

Comment:
caching this @timolegros
Does it impact any other place?
https://commonwealth.im/dashboard/chain-events

is it ok to refresh this every 5mins?